metabase/metabase
# Cleanup SAML settings in Admin

Epic https://github.com/metabase/metabase/issues/19428

Changes:
- SAML toggle was moved from the form to the authentication methods list
- When not enabled, the submit button text is `Save and enable`

How to test:
- Run Metabase EE
- Go to Admin -> Settings -> Authentication -> SAML
- Enter some url for for the provider URL
- Enter contents of `test_resources/sso/auth0-public-idp.cert` for the ceritifcate
- Make sure it is possible to submit the form

<img width="538" alt="Screenshot 2022-10-24 at 14 23 52" src="https://user-images.githubusercontent.com/8542534/197515069-d3378ee0-9b11-4095-8eba-6b9dfa28a145.png">
<img width="554" alt="Screenshot 2022-10-24 at 14 23 47" src="https://user-images.githubusercontent.com/8542534/197515095-90afa498-4828-48b8-8947-c3f7c1e84c90.png">
<img width="547" alt="Screenshot 2022-10-24 at 14 23 38" src="https://user-images.githubusercontent.com/8542534/197515105-bbd68cc6-8855-45a2-ac28-ddea7d290019.png">
